cmd/mesh/profile-dump.go

```diff
@@ -173,6 +174,33 @@ func genICPS(inFilename, profile, setOverlayYAML string, force bool, l *logger)
 	return finalYAML, finalICPS, nil
 }
 
+func genOverlayICPS(filename string) (string, *v1alpha2.IstioControlPlaneSpec, error) {
+	if strings.TrimSpace(filename) == "" {
```

Comment:
Good idea to do this but could you move it higher in the call stack so all functions get the trimmed version?

Reply:
Fixed.